src/taco-cli/package.json

```diff
@@ -54,7 +54,7 @@
         "gulp": "^3.8.10",
         "gulp-concat": "^2.4.2",
         "mocha": "latest",
-        "should": "*"
+        "should": "4.3.0"
```

Comment:
I changed it to 4.3 since that is what all our other packages used at the moment (and it matched the typescript definitions we have). I had a look at the later versions, and we should be compatible with 6.X I believe, but 7.x has made breaking changes where instead of doing `foo.should.be.true` you need to do `foo.should.be.true()`. I didn't want to go change everything without discussing things first, but I did want to fix this.